psp/scripts/eval_model.py

```diff
@@ -41,6 +41,7 @@
 )
 @click.option(
     "--eval-config",
+    "eval_config_path",
```

Comment:
@peterdudfield The destination of how the value is stored is set to eval_config_path here.

Reply:
if this wasnt stated then you would need to use eval_config

Reply:
ok, sounds good

Reply:
The rationale for using different names was that when calling the script, `--eval-config` is more succinct and clear enough; but once in the code, there is a difference between the path and the config itself. But `--eval-config-path` would work too.